marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/addMatchStepToFlow.spec.tsx

```diff
@@ -117,8 +117,8 @@ describe("Add Matching step to a flow", () => {
     runPage.deleteStep(matchStep, flowName1).click();
     loadPage.confirmationOptions("Yes").click();
 
-  });
-  it("Navigating to match tab", () => {
+    // });
+    // it("Navigating to match tab", () => {
```

Comment:
Hey, do we need this commented? Maybe we can just delete those lines

Reply:
For these fixes where I comment the transition into the next test, it's because of the flaky login issue where there would be issues logging into the next test. I thought it would be helpful to keep this comment in so you will know where the next test used to be, and when you replace the logins with local storage, we can add it back in. What do you think?

Reply:
Oh cool! Maybe we can include in the comment something mentioning that :)

Reply:
Pushed and left a comment there. Thanks @csilva-ml